Core/Entities/User.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.AspNetCore.Identity;
+namespace Core.Entities
+{
+    public class User : IdentityUser<Guid>
+    {
+        public User() => Id = Guid.NewGuid();
```

Comment:
feels weird given your entitybase

Reply:
can't inherit `EntityBase` because it has to inherit `IdentityUser`